### DISCUSSION TITLE
Video playback window progress bar

### DISCUSSION THREAD

**Discussion by darmach** [blakeblackshear/frigate#12918] [Ideas]
Please read my rationale behind this post before writing that we have the timeline ;)

Would it be possible to get progress/selection bar on the bottom of the video playback windows when in specific video review/playback window? I am aware that there are +- 10 second buttons and you can click on the timeline bar on the right, but I feel that the progress bar that you can click would be more useful. 

Most of the spreaming or video providers in the world display the video this way, whether you'd look on Youtube, Amazon or most streaming services. That bar often becomes second nature for internet users - I know for sure that my brain looks for it, and when I want to traverse the recording my first reflex is to move the cursor downwards toward the non-existing progress bar :)

Mind that it would not necessarily be doubling the functionality on the timeline on the right. The timeline allows to jump even between different recordings and would still act like time-of-the-day anchor to the moment you're watching :)

**Comment by NickM-27** [comment]
the problem is that a bar on the bottom doesn't make sense in this context. For example let's say you only have motion based recording and you are viewing 2:30 PM. The bar at the bottom can only possibly show the video length which would be between 2PM and 3PM. If there was only 15 minutes of motion during that hour then the bar at the bottom would show 0-15 minutes but it is not clear to the user at all what parts of that hour 2-3 PM make up that 15 minutes. If you want to jump to 2:45 there is no easy way to know what part of that bar actually corresponds to 2:45.

Using the timeline makes it very clear to the user what time of the video is being watching and what part of the video they are trying to move to. We already have plans for more tools to have better control of finely seeking on mobile and desktop
